src/kili/queries/asset/__init__.py

```diff
@@ -220,9 +220,20 @@ def assets(
             },
         }
 
-        post_call_process = get_post_assets_call_process(
-            download_media, local_media_dir, project_id
-        )
+        if download_media:
```

Comment:
this starts to be a big method, so hiding the code added below in `get_post_assets_call_process` could be a good idea